2021/08.md

```diff
@@ -78,13 +78,15 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |:-:|:-----:|:-------:|-------|-----------|
     |   | 3     | 10m     | [Relative indexing `.at()` method](https://github.com/tc39/proposal-relative-indexing-method) for Stage 4 | Shu-yu Guo |
     |   | 1     | 30m     | [Change Array by copy](https://github.com/tc39/proposal-change-array-by-copy) for Stage 2 ([spec](https://tc39.es/proposal-change-array-by-copy/), [slides-tbc](#)) | Robin Ricard & Ashley Claymore |
+    |   | 1     | 45m     | [Operator overloading](https://github.com/tc39/proposal-operator-overloading): Where should we go from here? | Daniel Ehrenberg |
     |   | 0     | 30m     | [`Array.asyncFrom`](https://github.com/js-choi/proposal-array-async-from) for Stage 1 ([spec](https://jschoi.org/21/es-array-async-from/), [slides](https://docs.google.com/presentation/d/1sbnssKjUjt732EQYGWHljJ0pWhA7DTCg-lnLB7IOpck/edit#slide=id.gc6fa3c898_0_16)) | J. S. Choi
 
 1. Longer or open-ended discussions
 
     | ✓ | timebox | topic | presenter |
     |:-:|:-------:|-------|-----------|
     |   | 60m     | Tightening host restrictions to improve testing ([example](https://github.com/tc39/test262/pull/3054#issuecomment-882741949)) | Jordan Harband |
+    |   | 30m     | WebAssembly/JS interaction and corresponding features | Daniel Ehrenberg |
```

Comment:
The PR title makes me think "oh Operator overloading will interact with WASM" 🤣🤣